### PR TITLE
lower error bound and cooldown mechanism

### PR DESCRIPTION
**Pull request by teisvh**

### CODE REVIEW

File: src/managers/BudgetManager.cpp [hunk 1]
```diff
@@ -184,6 +211,42 @@ void BudgetManager::increaseErrorBounds(){
...
+
+    }
+
+}
+
+void BudgetManager::lowerErrorBounds(int locID){
+
+    // Set error bound to 0
+    configManager.timeseriesCols.at(locID).error = 0;
+}
```
Comment (karljohan12) [inline src/managers/BudgetManager.cpp:252]: Flush model

File: src/managers/BudgetManager.cpp [hunk 2]
```diff
@@ -184,6 +211,42 @@ void BudgetManager::increaseErrorBounds(){
...
+    int stop = std::min(10, static_cast<int>(smalllestImpacts.size()-1));
+
+    for(int i = 0; i < stop; i++){
+
+        //Create config instance for adjusted modeManager
+        int locID = smalllestImpacts.at(i).second.first;
+
+        // Set error bound to 0
+        configManager.timeseriesCols.at(locID).error = 0;
+
```
Comment (karljohan12) [inline src/managers/BudgetManager.cpp]: Flush models

**Review by karljohan12 — CHANGES_REQUESTED** [review]

File: src/managers/BudgetManager.cpp (hunk 1 above)
Comment (teisvh) [reply src/managers/BudgetManager.cpp:252]: fixd

File: src/managers/BudgetManager.cpp (hunk 2 above)
Comment (teisvh) [reply src/managers/BudgetManager.cpp]: fixd

**Pull request merged**